src/flow-bypass.c

```diff
@@ -32,6 +32,13 @@
 
 #define FLOW_BYPASS_DELAY       10
 
+#ifndef TIMEVAL_TO_TIMESPEC 
```

Comment:
useless for now I agree, but it could be possibly implemented later by some capture methods. As the functions are modular, this could be useful in the future.